test/integration/rules/meta-viewport/meta-viewport-pass3.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta id="mvp" name="viewport" content="maximum-scale=2;" />
+  </head>
+
+  <body>
+    <div id="mocha"></div>
+    <!-- iframe with zooming disabled should not fail the rule-->
+    <iframe
```

Comment:
Unfortunately, IE doesn't support `srcdoc`. You'll have to add axe.js to the frame content, otherwise axe won't run there.